templates/extensions/foundry/README.md

```diff
@@ -0,0 +1,80 @@
+# 🏗 Scaffold-ETH 2
```

Comment:
Maybe the common parts between both READMEs can be in another file, and when the extension is installed, the README is merged. But maybe it's too much and we can duplicate these texts and that is.

Reply:
Good point! But not sure it's easy to do with .md files and with multiple params